spec/factories/arf_gi_bills.rb

```diff
@@ -4,10 +4,7 @@
   factory :arf_gi_bill do
     institution { 'SOME SCHOOL' }
     facility_code { generate :facility_code }
-
-    sequence :gibill do |n|
-      n
-    end
+    gibill { Random.rand(1000) }
```

Comment:
Is there a reason for using random values here?  I would usually advise against it because it's harder to find failed test cases again and it makes it possible (but unlikely) to have a repeat value that might cause surprising test issues.

Reply:
reverted